### DISCUSSION TITLE
GIT support

### DISCUSSION THREAD

**Discussion by fedetango** [phcode-dev/phoenix#697] [Ideas]
There are libraries that allow you to use git completely from the browser. This is the case for "isomorphic-git".

It would be great to work with my GIT repositories (not GitHub) from the application.

I'd be happy if at first you could at least open a Git project without the need to be GitHub.

**Comment by abose** [comment]
Tracked as part of https://github.com/phcode-dev/phoenix/issues/145 
Will be working on it post live preview and extension store.
